authorizer-app/src/app/components/rest-source-authorization/rest-source-user-list-reset-dialog.html

```diff
@@ -1,6 +1,6 @@
 <h1 mat-dialog-title>Reset User in project {{data.projectId}}?</h1>
 <div mat-dialog-content>
-  <p>This will pull all the data for the user (project: {{data.projectId}}, user-id: {{data.userId}} with external-user-id: {{data.externalUserId}}) again in the specified date range.</p>
+  <p>This will pull all the data for the user (project: {{data.projectId}}, user-id: {{data.externalId? data.externalId : data.userId}} with external-user-id: {{data.serviceUserId}}) again in the specified date range.</p>
```

Comment:
If the external user ID is shown, it would still be helpful to see the actual user ID between brackets, i.e. `blootsvoets (abcde-aefa)`